tests/unit/ml_handlers/test_openai.py

```diff
@@ -205,11 +219,11 @@ def test_prompt_template(self, mock_handler):
 
     @patch("mindsdb.integrations.handlers.postgres_handler.Handler")
     def test_bulk_normal_completion(self, mock_handler):
-        """ Tests normal completions (e.g. text-davinci-003) with bulk joins that are larger than the max batch_size """
+        """Tests normal completions (e.g. text-davinci-003) with bulk joins that are larger than the max batch_size"""
         # create project
         self.run_sql("create database proj")
-        handler_instance = OpenAIHandler()
-        N = 1 + handler_instance.max_batch_size  # get N larger than default batch size
+        max_batch_size = 10
```

Comment:
How would the test control that this number is actually bigger than what the handler is able to, ahm, handle? Say for instance OpenAI changes the maximum batch size from its current value (`N=20` I believe), this test should ideally test for `N=21`.

I guess we don't control this at the handler itself, but it seems like this change would introduce an additional parameter to control in order for the test to accomplish its objective.